modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/internal/upgrade/v4_1_0/DDMFieldUpgradeProcess.java

```diff
@@ -131,6 +133,7 @@ protected void doUpgrade() throws Exception {
 
 			while (resultSet.next()) {
 				_upgradeDDMContent(
+					resultSet.getLong("classNameId"),
 					resultSet.getLong("companyId"),
 					resultSet.getLong("contentId"),
 					insertDDMFieldPreparedStatement,
```

Comment:
@IstvanD Since `DDL` does not use Data Engine in 7.4.x (Master), do we really need to call `_upgradeDDMContent` for `DDLRecordSets`? 

@marcelabc what do you think?

Reply:
Hi @carolmariaabb DDL Record Sets are not created with Data Engine's editor, but they are nevertheless stored by the Data Engine framework eg. the DDMField, and DDMFieldAttribute table.